crates/serpen/tests/fixtures/comprehensive_ast_rewrite/core/__init__.py

```
@@ -0,0 +1,6 @@
+# Core package initialization
+from .utils.helpers import Logger as CoreLogger, process as core_process
+
+# Package-level conflicts
+result = "core_package_result"
```

Pylint (3.3.7)

[convention] Constant name "result" doesn't conform to UPPER_CASE naming style

(C0103)